app/js/components/manageApps/Addon.jsx

```diff
@@ -70,7 +70,8 @@ class Addon extends Component {
       .then(response => {
         this.getAffectedModules(response.data.uuid);
         return axios.get(`${ApiHelper.getAddonUrl()}?modulePackage=${response.data.packageName}`)
-          .then(response => {
+          .then(maintainers => {
```

Comment:
Looks good to me.
But i think a better alternative way to achieve this is to only change the instance of `uuid` to `uid` on line `383` and `487`.

Reply:
@Charpell the second response data overrides the initial response data. This was one of the main causes for the uuid not being found.

The uuid is important for starting and stopping the module